MapboxNavigation-SPM.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -51,26 +51,26 @@
         "repositoryURL": "https://github.com/mapbox/mapbox-events-ios.git",
         "state": {
           "branch": null,
-          "revision": "8e0c725271dee0a0c957b9b9738aef81b601f12b",
-          "version": "1.0.2"
+          "revision": "e4ded10f5b9594374416f9396d5b540f8c491bbe",
+          "version": "1.0.3"
         }
       },
       {
         "package": "MapboxMaps",
         "repositoryURL": "https://github.com/mapbox/mapbox-maps-ios.git",
         "state": {
           "branch": null,
-          "revision": "5b8b0246f283e87fb98193e67f7bf8eb0a490e1f",
-          "version": "10.0.0-rc.7"
+          "revision": "bedc49744970602ebc10c6de3f8a2b4cc6f084a6",
+          "version": "10.0.0-rc.8"
         }
       },
       {
         "package": "MapboxNavigationNative",
         "repositoryURL": "https://github.com/mapbox/mapbox-navigation-native-ios.git",
         "state": {
           "branch": null,
-          "revision": "9fd85d49b91be345333c352016ea3b7b48db2bc1",
-          "version": "65.0.2"
+          "revision": "7c8ae18aa21bea2b4e801f0b6aab6a2a21d4aec1",
+          "version": "66.0.0"
```

Comment:
v66.0.1 is now available. No need to make it a strict requirement, but the resolved files can be updated.